### PR TITLE
Rename transporation recovery to traffic flow recovery

### PR DESCRIPTION
**Pull request by ywkim312**

### CODE REVIEW

File: pyincore/analyses/transportationrecovery/transportationrecovery.py [hunk 1]
```diff
@@ -1,304 +1,19 @@
-from __future__ import division
-import pandas as pd
-import copy
-import random
+# This program and the accompanying materials are made available under the
```
Comment (longshuicy) [inline pyincore/analyses/transportationrecovery/transportationrecovery.py:1]: Could you add the copyright year here. I recall this analysis is from 2018.

**Review by longshuicy — APPROVED** [review]

**Comment by navarroc** [comment]
@ywkim312 Please add the copyright year to the files and then I think this can be merged. Thanks!

File: pyincore/analyses/transportationrecovery/transportationrecovery.py (hunk 1 above)
Comment (ywkim312) [reply pyincore/analyses/transportationrecovery/transportationrecovery.py:1]: Added and used as 2018

**Comment by ywkim312** [comment]
> @ywkim312 Please add the copyright year to the files and then I think this can be merged. Thanks!

added

File: pyincore/analyses/transportationrecovery/transportationrecovery.py (hunk 1 above)
Comment (longshuicy) [reply pyincore/analyses/transportationrecovery/transportationrecovery.py:3]: Did you push? Sorry I don't see it gets reflected.

**Pull request merged**